README.md

```diff
@@ -8,6 +8,12 @@
 [![Gitter](https://badges.gitter.im/covid19-review/community.svg)](https://gitter.im/covid19-review/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 <!-- usage note: delete CI badges above for services not used by your manuscript -->
 
+## Code of Conduct
+
+Participation in this project requires that you adhere to the [code of conduct](CODE_OF_CONDUCT.md).
```

Comment:
This could be a place to indicate how contributors are expected to sign the code of conduct.

Reply:
If @agitter and @cgreene think 038d116 looks good, we should be OK to merge